src/ScottPlot.Demo.Avalonia/AvaloniaDemos/PlotViewer.axaml

```diff
@@ -0,0 +1,64 @@
+﻿<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+		Width="600"
+		Height="400"
+        x:Class="ScottPlot.Demo.Avalonia.AvaloniaDemos.PlotViewer"
+        Title="PlotViewer">
+	<DockPanel Margin="10">
+		<TextBlock TextWrapping="Wrap" DockPanel.Dock="Top">
+			The PlotViewer lets you launch a plot in an interactive pop-up window.
+		</TextBlock>
+		<TextBlock TextWrapping="Wrap" DockPanel.Dock="Top">
+			You can focus on generating interesting data, and PlotViewer will handle the GUI!
+		</TextBlock>
+
+		<StackPanel Orientation="Horizontal" DockPanel.Dock="Top">
+
+			<Border Margin="20">
+				<StackPanel Orientation="Horizontal">
+					<TextBlock Text="Random Walk Generator"/>
+					<TextBlock Text="Points:" VerticalAlignment="Center"/>
+					<TextBox x:Name="WalkPointsTextbox" Text="100" Width="35" VerticalAlignment="Center"/>
+					<Button Content="Launch" Padding="5, 3, 5, 3" Margin="5" Click="LaunchRandomWalk"/>
+				</StackPanel>
+			</Border>
+
+			<Border Margin="20">
+				<StackPanel Orientation="Horizontal">
+					<TextBlock Text="Sine Wave Generator" />
+					<TextBlock Text="Waves:" VerticalAlignment="Center"/>
+					<TextBox x:Name="SinCountTextbox" Text="3" Width="35" VerticalAlignment="Center"/>
+					<Button Content="Launch" Padding="5, 3, 5, 3" Margin="5" Click="LaunchRandomSin"/>
+				</StackPanel>
+			</Border>
+
+		</StackPanel>
+
+		<Border>
+			<StackPanel>
+				<TextBlock Text="Sample Code"/>
+				<ScrollViewer HorizontalScrollBarVisibility="Auto">
+					<StackPanel>
+						<!-- I like Avalonia, but not allowing newlines in TextBlocks isn't quirky or cute... -->
```

Comment:
Avalonia does support multiline TextBlock

Reply:
My issue was actually the handling of tabs on newlines, not the newline characters. I was also a little too snarky, I'm sorry.